README.md

```diff
@@ -31,6 +31,19 @@ $ export AUTH_USERNAME="vault"
 $ export AUTH_PASSWORD="broker-secret-password"
 ```
 
+### Creating a Dev Environment
+
+Note: This section takes a good amount of time (likely, an hour or more) due to downloading a 16gb ISO and VirtualBox taking a while to create an environment.
+
+1. Register for an account at `https://pivotal.io`.
+2. Install the `cf` CLI: https://docs.cloudfoundry.org/cf-cli/install-go-cli.html.
+3. Ensure you have VirtualBox and Java 7 installed.
```

Comment:
Is any JRE ok? (e.g. OpenJDK vs Oracle?

Reply:
Actually, as I'm re-reading the installation instructions, Java may only be required to build the sample app. I'll strip the note about requiring Java.